### PR TITLE
Add the Allow tags option for parental control documentation

### PR DESCRIPTION
**Pull request by sa2000a**
We document blocking content option using tags but not mention the reverse option that is provided for allowing only content with specific tags.

This PR adds the "Allow only items with these tags" to our articles

This change comes from request from GrimReaper in this forum discussion https://emby.media/community/index.php?/topic/131089-user-specific-media-lists/&do=findComment&comment=1376549

### CODE REVIEW

**Comment by sa2000a** [comment]
Thank You @LukePulverenti 

I uploaded a second commit which got picked up in the PR - I meant to raise a separate PR for it. It is now merged as well - I should have waited until the parental control change was processed

so just adding here some retrospective documentation for the 2nd commit:

It follows from request here from Happy2Play https://emby.media/community/index.php?/topic/130323-emby-does-not-display-lrc-lyrics/#comment-1379851

Basically we had a paragraph about Skip Intro support on LG TV in the Music Lyrics article. The commit to fix this is 
https://github.com/sa2000a/Emby.Docs/commit/4c861c5e6bea0cdeb4bbe23d75812cf3a3832181
